### DISCUSSION TITLE
Get current position of camera in 3D scene

### DISCUSSION THREAD

**Discussion by thaije** [zauberzeug/nicegui#3459] [Ideas / Feature Requests]
### Question

Hey all, 

Has anyone figured out if it is possible to get the correct current camera position from a 3D scene? 
It is possible to change the camera with [`scene.move_camera()`](https://nicegui.io/documentation/scene#wait_for_initialization), and also get the position using [`scene.camera`](https://github.com/zauberzeug/nicegui/blob/main/nicegui/elements/scene.py#L105), but the positions from the camera aren't updated when the user drags the camera around in the scene it seems. 

We are using the 3D scene to visualize a robot that autonomously explores an environment with detections being added as objects in the scene. When the user zooms in or out I would like to dynamically scale the objects in the 3D scene so they remain properly visible and clickable.  

If anyone has any leads or tips on how to fix this, let me know! :smile: 

# minimal code example

``` python
from nicegui import ui

threedscene = ui.scene(background_color='#222')
with threedscene.classes('w-full h-64') as scene:
    scene.box()

def check_cam():
    print(threedscene.camera)

ui.timer( 
    interval=1,
    callback=check_cam
)

ui.run()

# Do some dragging of the camera
```

**Comment by falkoschindler** [comment]
Hi @thaije,

I don't think this is possible at the moment. But I created PR #3465 to add a new `get_camera` method.

**Comment by thaije** [comment]
Thanks @falkoschindler for the quick reply and pr. This would indeed solve the issue! :smile:
